src/contacts/ViewContact/ViewContactContainer.js

```diff
@@ -87,17 +94,18 @@ class ViewContactContainer extends Component {
     const { showConfirmDelete, showConfirmUnassign } = this.state;
 
     const contact = get(resources, 'contact.records[0]', {});
-    const categories = get(resources, 'categories.records', []);
+    const contactCategories = get(resources, `${DICT_CATEGORIES}.records`, []);
```

Comment:
I believe it should be something like `const contactCategories = get(resources, `${DICT_CATEGORIES}.records`);`
and below `if (get(resources, 'contact.isPending', true) || !contactCategories`

Reply:
Fixed